.gitlab-ci.yml

```diff
@@ -113,6 +144,19 @@ agent_base_image:
     - scheduled
     - /.*ci-base-image.*/
 
+host_client_base_image:
+  stage: docker
+  image: docker:19.03.1
+  services:
+    - docker:19.03.1-dind
+  script:
+    - docker login -u gitlab-ci-token -p $CI_BUILD_TOKEN ${CI_REGISTRY}
+    - docker build -t ${host_client_image} --build-arg CI_REGISTRY=${CI_REGISTRY} -f micro_ros_setup/docker/Dockerfile.host_client .
+    - docker push ${host_client_image}
+  only:
+    - scheduled
```

Comment:
Should by "schedules" -- I also made this mistake in some earlier versions, but current crystal should be correct.